NRasterizer.CLI/Program.cs

```diff
@@ -120,11 +134,17 @@ public static void Main(string[] args)
             var rasterizerName = args[0];
             var fontPath = new FileInfo(args[1]);
             var target = new FileInfo(args[2]);
-            var text = args[3];
+            var text = args[3].Replace("\\n", "\n").Replace("\\t", "\t");
```

Comment:
:+1: This could even go instatic function in `Program` - `ExpandWhitespace` or `UnescapeWhitespace` to make it even clearer what it does.